templates/tools/dockerfile/grpc_artifact_python_musllinux_1_2_aarch64/Dockerfile.inja

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# AUTO-GENERATED FROM `$REPO_ROOT/templates/tools/dockerfile/grpc_artifact_python_musllinux_1_1_aarch64/Dockerfile.inja`!!!
+# AUTO-GENERATED FROM `$REPO_ROOT/templates/tools/dockerfile/grpc_artifact_python_musllinux_1_2_aarch64/Dockerfile.inja`!!!
 
-FROM quay.io/pypa/musllinux_1_1_aarch64:2024-09-09-f386546
+FROM quay.io/pypa/musllinux_1_2_aarch64:2025.07.27-1
```

Comment:
This should be done separately, but we should bump this to `2025.08.02-1` (or later) to take care of the most recent setuptools CVE: https://nvd.nist.gov/vuln/detail/CVE-2025-47273